packages/tracing/src/index.ts

```diff
@@ -1,47 +1,261 @@
-  spanStatusfromHttpCode,
-  startIdleTransaction,
-  stripUrlQueryAndFragment,
+import type {
+  RequestInstrumentationOptions as RequestInstrumentationOptionsT,
+  SpanStatusType as SpanStatusTypeT,
 } from '@sentry-internal/tracing';
-export type { RequestInstrumentationOptions, SpanStatusType } from '@sentry-internal/tracing';
-
 import {
-  addExtensionMethods,
+  addExtensionMethods as addExtensionMethodsT,
   Apollo,
-  BrowserTracing,
+  BROWSER_TRACING_INTEGRATION_ID as BROWSER_TRACING_INTEGRATION_ID_T,
+  BrowserTracing as BrowserTracingT,
+  defaultRequestInstrumentationOptions as defaultRequestInstrumentationOptionsT,
   Express,
+  extractTraceparentData as extractTraceparentDataT,
+  getActiveTransaction as getActiveTransactionT,
   GraphQL,
+  hasTracingEnabled as hasTracingEnabledT,
+  IdleTransaction as IdleTransactionT,
+  instrumentOutgoingRequests as instrumentOutgoingRequestsT,
   Mongo,
   Mysql,
   Postgres,
   Prisma,
+  Span as SpanT,
+  // eslint-disable-next-line deprecation/deprecation
+  SpanStatus as SpanStatusT,
+  spanStatusfromHttpCode as spanStatusfromHttpCodeT,
+  startIdleTransaction as startIdleTransactionT,
+  stripUrlQueryAndFragment as stripUrlQueryAndFragmentT,
+  TRACEPARENT_REGEXP as TRACEPARENT_REGEXP_T,
+  Transaction as TransactionT,
 } from '@sentry-internal/tracing';
 
+// BrowserTracing is already exported as part of `Integrations` below (and for the moment will remain so for
+// backwards compatibility), but that interferes with treeshaking, so we also export it separately
+// here.
+/**
+ * @deprecated `@sentry/tracing` has been deprecated and will be removed in the next major version.
+ * `BrowserTracing` can be imported from `@sentry/browser` or your framework SDK
+ *
+ * import { BrowserTracing } from '@sentry/browser';
+ * new BrowserTracing()
+ */
+export const BrowserTracing = BrowserTracingT;
+
+// BrowserTracing is already exported as part of `Integrations` below (and for the moment will remain so for
+// backwards compatibility), but that interferes with treeshaking, so we also export it separately
+// here.
+/**
+ * @deprecated `@sentry/tracing` has been deprecated and will be removed in the next major version.
+ * `BrowserTracing` can be imported from `@sentry/browser` or your framework SDK
+ *
+ * import { BrowserTracing } from '@sentry/browser';
+ * new BrowserTracing()
+ */
+export type BrowserTracing = BrowserTracingT;
+
+/**
+ * @deprecated `@sentry/tracing` has been deprecated and will be removed in the next major version.
+ */
+export const addExtensionMethods = addExtensionMethodsT;
+
+/**
+ * @deprecated `@sentry/tracing` has been deprecated and will be removed in the next major version.
+ *
+ * `getActiveTransaction` can be imported from `@sentry/node`, `@sentry/browser`, or your framework SDK
+ */
+export const getActiveTransaction = getActiveTransactionT;
+
+/**
+ * @deprecated `@sentry/tracing` has been deprecated and will be removed in the next major version.
```

Comment:
Can we replace all of these:

> @deprecated `@sentry/tracing` has been deprecated and will be removed in the next major version.

with e.g.:


> @deprecated `@sentry/tracing` has been deprecated and will be moved to to @sentry/node, @sentry/browser, or your framework SDK in the next major version.

To make it clear immediately that this will not be _removed_.